paddle/fluid/operators/conv_op.cc

```diff
@@ -412,18 +415,45 @@ framework::OpKernelType ConvOpGrad::GetExpectedKernelType(
                                  customized_type_value);
 }
 
+class Conv2dGradMaker : public framework::SingleGradOpDescMaker {
+ public:
+  using framework::SingleGradOpDescMaker::SingleGradOpDescMaker;
+
+  std::unique_ptr<framework::OpDesc> Apply() const override {
+    auto* op = new framework::OpDesc();
+    op->SetType(GradOpType());
+    op->SetInput("Input", Input("Input"));
+    op->SetInput("Filter", Input("Filter"));
+    op->SetInput("Bias", Input("Bias"));
+    // ResidualData is not used in conv2d_grad ops, we comment this temporarily
+    // op->SetInput("ResidualData", Input("ResidualData"));
```

Comment:
if "ResidualData" is not used in grad op, it will be deleted after forward op finished. Our memory optimize will do that automatically. I think you can remove this comment safely.

Reply:
removed